contrib/sacrebleu/sacrebleu.py

```diff
@@ -170,7 +176,7 @@
 import math
 import unicodedata
 
-VERSION = '1.2.3'
+VERSION = '1.2.5'
```

Comment:
What happened to 1.2.4?

Reply:
I was a bit hasty in pushing up to pypi, which doesn't let you revert a version number once it's pushed.